[2025-05-19T09:29:42.598+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nyc_taxi_pipeline.ingest_data manual__2025-05-19T09:29:40.855638+00:00 [queued]>
[2025-05-19T09:29:42.609+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nyc_taxi_pipeline.ingest_data manual__2025-05-19T09:29:40.855638+00:00 [queued]>
[2025-05-19T09:29:42.610+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-19T09:29:42.628+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ingest_data> on 2025-05-19 09:29:40.855638+00:00
[2025-05-19T09:29:42.633+0000] {standard_task_runner.py:60} INFO - Started process 182 to run task
[2025-05-19T09:29:42.639+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'nyc_taxi_pipeline', 'ingest_data', 'manual__2025-05-19T09:29:40.855638+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/nyc_taxi_pipeline.py', '--cfg-path', '/tmp/tmp9pwzif_l']
[2025-05-19T09:29:42.642+0000] {standard_task_runner.py:88} INFO - Job 38: Subtask ingest_data
[2025-05-19T09:29:42.662+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-19T09:29:42.706+0000] {task_command.py:423} INFO - Running <TaskInstance: nyc_taxi_pipeline.ingest_data manual__2025-05-19T09:29:40.855638+00:00 [running]> on host 02ed670443a2
[2025-05-19T09:29:42.987+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nyc_taxi_pipeline' AIRFLOW_CTX_TASK_ID='ingest_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T09:29:40.855638+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-19T09:29:40.855638+00:00'
[2025-05-19T09:29:42.989+0000] {nyc_taxi_pipeline.py:23} INFO - Starting data ingestion...
[2025-05-19T09:29:42.993+0000] {ingest.py:20} INFO - File already exists at /opt/***/scripts/Data/Raw/data.parquet. Skipping dowload.
[2025-05-19T09:29:42.994+0000] {nyc_taxi_pipeline.py:31} INFO - Data ingestion complete.
[2025-05-19T09:29:42.996+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-19T09:29:43.006+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=nyc_taxi_pipeline, task_id=ingest_data, execution_date=20250519T092940, start_date=20250519T092942, end_date=20250519T092943
[2025-05-19T09:29:43.055+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-19T09:29:43.095+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
